client/settings/payment-request-section/index.js

```diff
@@ -76,176 +83,181 @@ const PaymentRequestSection = () => {
 		wc_stripe_settings_params.is_amazon_pay_available && // eslint-disable-line camelcase
 		availablePaymentMethodIds.includes( PAYMENT_METHOD_AMAZON_PAY );
 
+	const isAmazonPayDisabled =
+		amazonPayUnavailableReason !== null &&
+		! enabledMethodIds.includes( PAYMENT_METHOD_AMAZON_PAY );
+
+	const isApplePayGooglePayDisabled =
+		applePayGooglePayUnavailableReason !== null &&
+		! isPaymentRequestEnabled;
+
+	const isLinkDisabled =
+		linkUnavailableReason !== null && ! isStripeLinkEnabled;
+
 	return (
 		<Card className="express-checkouts">
 			<CardBody size={ 0 }>
 				<ul className="express-checkouts-list">
-					{ ! displayExpressPaymentMethods &&
-						! displayLinkPaymentMethod && (
-							<li className="express-checkout">
-								<div>
-									{ __(
-										'Credit card / debit card must be enabled as a payment method in order to use Express Checkout.',
-										'woocommerce-gateway-stripe'
-									) }
-								</div>
-							</li>
-						) }
-					{ ! displayExpressPaymentMethods &&
-						displayLinkPaymentMethod && (
-							<li className="express-checkout">
-								<div>
-									{ __(
-										'Credit card / debit card must be enabled as a payment method in order to use Google Pay and Apple Pay (and Link in the classic checkout).',
+					<li className="express-checkout has-icon-border">
+						<div className="express-checkout__checkbox">
+							<CheckboxControl
+								label={ __(
+									'Apple Pay / Google Pay Input',
+									'woocommerce-gateway-stripe'
+								) }
+								checked={ isPaymentRequestEnabled }
+								onChange={ updateIsPaymentRequestEnabled }
+								disabled={ isApplePayGooglePayDisabled }
+							/>
+						</div>
+						<div className="express-checkout__icon">
+							<PaymentRequestIcon size="medium" />
+						</div>
+						<div className="express-checkout__label-container">
+							<div className="express-checkout__label">
+								{ __(
+									'Apple Pay / Google Pay',
+									'woocommerce-gateway-stripe'
+								) }
+								<PaymentMethodRequiresCardMethodPill
```

Comment:
[nit] I prefer conditionally rendering elements instead of having components return null. I don't like the idea of mounting/unmounting components that won't render anything. But I agree the overload is minimal, as we have only 2 usages in the entire page, so just an opinion 🤷🏼 

Moving this logic to this component:
```javascript
const unavailableReason = usePaymentMethodUnavailableReason( PAYMENT_METHOD_APPLE_PAY_GOOGLE_PAY );
const showRequiresCardMethodPill = unavailableReason === PAYMENT_METHOD_UNAVAILABLE_REASONS.REQUIRES_CARD_METHOD;
```
and then doing:
```javascript
{ showRequiresCardMethodPill && <PaymentMethodRequiresCardMethodPill ... }
```